docs/config-file/v2.rst

```diff
@@ -0,0 +1,365 @@
+
+.. code-block:: yaml
+   
+   version: 2
+
+.. warning::
+   
+   If you don't provide the version, :doc:`v1 <v1>` would be used.
+
+formats
+~~~~~~~
+
+Formats of the documentation to be built.
+
+:Type: ``list``
+:Options: ``htmlzip``, ``pdf``, ``epub``
+:Default: ``[]``
+
+.. code-block:: yaml
+
+   # Default
+   formats: []
+
+.. code-block:: yaml
+
+   # Build PDF & ePub
+   formats:
+     - epub
+     - pdf
+
+.. note::
+
+   You can use the ``all`` keyword to indicate all formats.
+
+   .. code-block:: yaml
+
+      # Build all formats
+      formats: all
+
+python
+~~~~~~
+
+Configuration of the Python environment to be used.
+
+.. code-block:: yaml
+
+   python:
+      version: 3.6
+      requirements: docs/requirements.txt
+      install: pip
+      extra_requirements:
+        - docs
+      system_packages: true
+
+python.version
+``````````````
+
+The Python version (this depends on :ref:`config-file/v2:build.image`).
+
+:Type: ``number``
+:Default: ``3``
+
+python.requirements
+```````````````````
+
+The path to the requirements file from the root of the project.
+
+:Type: ``path``
+:Default: ``null``
+
+If the value is ``null``,
+Read the Docs would try to find a requirements file in your ``docs/`` directory and in the root of your project.
+If you want to completely ignore the requirements file, you can use an empty string ``''``.
+
+python.install
+``````````````
+
+Install the project using python ``python setup.py install`` or ``pip install .``.
+
+:Type: ``string``
+:Options: ``pip``, ``setup.py``
+:Default: ``null``
+
+If the value is ``null``, Read the Docs would not install your project.
+
+python.extra_requirements
+`````````````````````````
+
+`Extra requirements <https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies>`_
+section to install in addition to the `package dependencies <https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies>`_.
+
+:Type: ``list``
+:Default: ``[]``
+
+.. warning::
+
+   You need to install your project with ``pip`` to use this feature.
+   See :ref:`config-file/v2:python.install`.
+
+Behind the scene the following Pip command will be run::
+
+   $ pip install -e .[tests,docs]
+
+python.system_packages
+``````````````````````
+
+Give the virtual environment access to the global site-packages directory.
+
+:Type: ``bool``
+:Default: ``false``
+
+conda
+~~~~~
+
+Configuration for Conda support.
+
+.. code-block:: yaml
+
+   conda:
+     environment: environment.yml
+
+conda.environment
+`````````````````
+
+The path to the Conda environment file from the root of the project.
+
+:Type: ``path``
+:Required: ``true``
+
+build
+~~~~~
+
+Configuration for the documentation build process.
+
+.. code-block:: yaml
+
+   build:
+     image: latest
+
+   python:
+     version: 3.6
+
+
+build.image
+```````````
+
+The build Docker image to be used.
+
+:Type: ``string``
+:Options: ``stable``, ``latest``
```

Comment:
We are not saying which is the default value here.

Reply:
Good catch!